proto/catalog.proto

```diff
@@ -33,6 +33,16 @@ message Source {
   string owner = 7;
 }
 
+message Sink {
+  uint32 id = 1;
+  uint32 schema_id = 2;
+  uint32 database_id = 3;
+  string name = 4;
+  string mv_name = 5;
```

Comment:
```suggestion
  uint32 associated_table_id = 5;
```

Reply:
> 

Thank you for this suggestion, I will implement it in my next PR :)